src/meta/MetaServiceUtils.h

```diff
@@ -19,9 +19,8 @@ enum class EntryType : int8_t {
     TAG         = 0x02,
     EDGE        = 0x03,
     USER        = 0x04,
-    TAG_INDEX   = 0x05,
-    EDGE_INDEX  = 0x06,
-    CONFIG      = 0x07,
+    INDEX       = 0x05,
+    CONFIG      = 0x06,
```

Comment:
Maybe you need to keep the CONFIG for 0x06, Otherwise, config will be lost? At least in some online environment.

Reply:
Really good cache:P